src/lib/rest/rest.cpp

```diff
@@ -1278,14 +1303,17 @@ void restInit
   const char* key  = _httpsKey;
   const char* cert = _httpsCertificate;
 
-  port          = _port;
-  restServiceV  = _restServiceV;
-  ipVersionUsed = _ipVersion;
-  serveFunction = (_serveFunction != NULL)? _serveFunction : serve;
-  acceptTextXml = _acceptTextXml;
-  multitenant   = _multitenant;
-  rushHost      = _rushHost;
-  rushPort      = _rushPort;
+  port             = _port;
```

Comment:
Almost never *only* indents.
The indents nirmally come as a new var has been introduced.

In this case, connMemory, maxConns and threadPoolSize